tests/integration-tests/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import server from "../../src/index";
+
+afterAll(async () => {
+  // workaround. It'd fails to kill server before jest finish its execution
+  // so it results in failure of jest to stop.
+  setTimeout(() => {
+    server.close();
+  }, 1000);
```

Comment:
This is pretty hacky, but I needed to wrap `server.close()` with in `setTimeout`. The problem was that jest kills the server even before the test suits finish executing, so eventually, it makes the test fail even though cases are passing.
If you know a clue or better solution, please lemme know as I'd be more than happy to fix that up

Reply:
You can create an issue for that if you want. we can come back later.